charts/gp-reporting-kafka-health/v0.0.1/questions.yml

```diff
@@ -0,0 +1,162 @@
+      - 10
+    default: "01"
+
+  ####
+  #### Reporting Kafka Health Service Settings
+  ####
+  - variable: build.commitSHA
+    type: string
+    required: true
+    group: "Application"
+    label: Application Build SHA
+    description: The commit SHA of the GitPrime Application build you want to deploy.
+    default: latest
+
+  - variable: volumes.enableLocalMount
+    type: boolean
+    required: true
+    group: "Application"
+    label: "Use Local Mount"
+    description: "If enabled, use the node's local storage directory for repository storage"
+    default: false
+    show_subquestion_if: "true"
+    subquestions:
+      - variable: volumes.nodeMountDirectory
+        type: string
+        label: "Directory on Node"
+        description: "The directory on the node to use for repository storage"
+        required: true
+        default: "/mnt/repo-storage"
+      - variable: volumes.podMountDirectory
+        type: string
+        label: "Directory on Pod"
+        description: "The directory inside the pod to mount the volume."
+        required: true
+        default: "/var/data"
+
+  - variable: datadog.enabled
+    type: boolean
+    required: true
+    group: "Application"
+    label: "Enable DataDog"
+    description: "Turn DataDog APM on for the pipeline"
+
+  - variable: dns.ndots
+    type: string
+    required: false
+    group: "Application"
+    label: "Number of ndots for DNS configuration"
+    description: "Modifies the number of ndots in /etc/resolv.conf"
+    default: 2
+
+  # Reporting Kafka Health Service Settings
+  - variable: application.webPort
+    type: string
+    required: true
+    group: "Application / Reporting Kafka Health Service"
+    label: Application port binding
+    description: The port binding that the application uses to listen to web requests
+    default: "8080"
+
+  - variable: application.nodePort
+    type: string
+    required: true
+    group: "Application / Reporting Kafka Health Service"
+    label: Application node port
+    description: The port that is exposed on the node which routes to the app port binding
+    default: "30680"
+
+  - variable: application.replicaCount
+    type: string
+    required: true
+    group: "Application / Reporting Kafka Health Service"
+    label: Number of Pods for services
+    description: The number of pods to deploy for the services that handle new repositories.
+    default: 1
+
+  - variable: application.javaOptions
+    type: string
+    required: true
+    group: "Application / Reporting Kafka Health Service"
+    label: Reporting Kafka Health Service Java Options
+    description: The Java options for the services.
+    default: "-d64 -Xms2g -Xmx4g"
+
+  - variable: application.resources.requests.cpu
+    type: string
+    required: true
+    group: "Application / Reporting Kafka Health Service"
+    label: Reporting Kafka Health Service Requested CPU
+    description: The number of CPUs the services will request.
+    default: "2"
```

Comment:
This is going to be a low=CPU app so I'd set this to 1/10 of a CPU, or "100m". More info about resources can be found here: https://kubernetes.io/docs/tasks/configure-pod-container/assign-cpu-resource/#specify-a-cpu-request-and-a-cpu-limit

Reply:
Set to 100 milliCPUs